types/d3-scale/index.d.ts

```diff
@@ -1286,7 +1287,7 @@ export function scaleSequential<Output = number>(interpolator?: (t: number) => O
  * @param interpolator The interpolator function to be used with the scale.
  */
 export function scaleSequential<Output>(
-    domain: [NumberValue, NumberValue],
+    domain: Iterable<NumberValue>,
```

Comment:
I really thought I'd already left this comment...  Maybe there was a similar PR to a different package?

Anyway, changing from a 2-tuple to an iterable seems like a downgrade - was this intentional?

Reply:
I don't know how to avoid it. They now accept iterables and there's no way to specify the length of the iterable, right?

Reply:
Disappointing, but I can't think of an alternative either.